src/js/components/common/dialogs/__snapshots__/deviceconnectiondialog.test.js.snap

```diff
@@ -76,7 +76,16 @@ exports[`DeviceConnectionDialog Component renders correctly 1`] = `
                 id="deb-package-tip"
               >
                 <p>
-                  The steps in the guide should work on most operating systems in the debian family (e.g. Debian, Ubuntu, Raspberry Pi OS) and devices based on ARMv6 or newer (e.g. Raspberry Pi 2/3/4, Beaglebone).
+                  The steps in the guide should work on most operating systems in the debian family (e.g. Debian, Ubuntu, Raspberry Pi OS) and devices based on ARMv6 or newer (e.g. Raspberry Pi 2/3/4, Beaglebone). Visit
```

Comment:
'debian family' should be 'Debian family', it's been the incorrect case for a while!